### ISSUE TITLE
3. Відсутність ідентифікаторів для елементів (Lack of Accessibility)

### ISSUE THREAD

**Issue by Ghost14883** [Ghost14883/Praktika_Labs#3]
Проблема:
У <label> відсутній атрибут for, що робить їх менш доступними для користувачів із технологіями підтримки.

Моє вирішення:
Додати id для input і прив’язати label через for, або замінити реалізацію на більш сучасну.

Ось повністю готовий код з вирішенням проблем

<!DOCTYPE html>
<html lang="uk">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Аккордеон Меню</title>
    <link rel="stylesheet" href="Style.css">
</head>
<body>

<h1>Аккордеон Меню</h1>

<details>
    <summary>Секція 1</summary>
    <p>Освіта — це ключ до успіху, що відкриває двері до нових можливостей та розширює горизонти нашого мислення.</p>
</details>

<details>
    <summary>Секція 2</summary>
    <p>Збалансоване харчування, регулярна фізична активність та позитивне мислення допомагають зберігати енергію і мотивацію.</p>
</details>

<details>
    <summary>Секція 3</summary>
    <p>Навчіться знаходити баланс між роботою та особистим життям. Час для себе — це не розкіш, а необхідність.</p>
</details>

</body>
</html>
